vbfml/scripts/err_analysis

```diff
@@ -378,5 +384,67 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     model.evaluate(sequence)
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i", "--input-dir", required=True, multiple=True, help="files to compare"
```

Comment:
I would suggest renaming this to something like `--input-dirs`, to highlight that there are multiple arguments to this. Also can you update the help text? This argument points to the directories right, so saying something like "Training directories to compare" could be more helpful.

Reply:
done